test/extended/networking.sh

```diff
@@ -28,7 +28,7 @@ DEFAULT_SKIP_LIST=(
   "\[Feature:Federation\]"
 
   # Skipped until https://github.com/openshift/origin/issues/11042 is resolved
-  "should preserve source pod IP for traffic thru service cluster IP"
+  "should preserve source pod IP for traffic through service cluster IP"
```

Comment:
These are literal names of tests to be skipped, and you did not change the name of the test. This is a correct misspelling but an incorrect change to the program.